pandevice/base.py

```diff
@@ -1537,6 +1537,31 @@ def delete_type(self, cType):
         for x in objs:
             self.remove(x)
 
+    def request_password_hash(self, value):
```

Comment:
It seems like this would be useful to call on a firewall object to get a password hash.  Should this method be moved to the `PanDevice` class?